wiki/de/de-Command-Line-Options.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: wiki
+title: "Kommandozeilen Optionen"
+lang: "en"
+permalink: "/wiki/Command-Line-Options"
+---
+
+# Kommandozeilen Optionen
+
+Zusätzlich zum setzen der Funktionen in der Jamulus Oberfläche kann man auch Funktionen über die Kommandozeile setzen. Die genaue Vorgehensweise hängt vom verwendeten Betriebssystem ab.
+
+Um zum Beispiel bei Windows die Rekorderfunktion mit dem `-R` Argument zu aktivieren : Rechtsklick auf die Jamulus Verknüpfung und wählen Sie "Eigenschaften" > Ziel. Füge dann die nötigen Argumente zu Jamulus.exe hinzu:
```

Comment:
"Klicke mit der rechten Maustaste auf [...] und wähle [...]" sounds more natural then "Rechtsklick auf die Jamulus Verknüpfung" for me.